### PR TITLE
WFLY-13655 LOG INFO level message if the Hibernate second level cache is enabled 

### PR DESCRIPTION
**Pull request by scottmarlow**
https://issues.redhat.com/browse/WFLY-13655

This is a minor change to show when the second cache is enabled.

### CODE REVIEW

**Review by kabir — APPROVED** [review]

File: jpa/hibernate5_3/src/main/java/org/jboss/as/jpa/hibernate5/JpaLogger.java [hunk 1]
```diff
@@ -31,8 +35,17 @@
...
      */
     JpaLogger JPA_LOGGER = Logger.getMessageLogger(JpaLogger.class, "org.jipijapa");
 
+    /**
+     * Inform that the Hibernate second level cache is enabled.
+     *
+     * @param puUnitName the persistence unit name
+     */
+    @LogMessage(level = INFO)
+    @Message(id = 20253, value = "second level cache enabled for %s")
```
Comment (jamezp) [inline jpa/hibernate5_3/src/main/java/org/jboss/as/jpa/hibernate5/JpaLogger.java:48]: This should be "Second", upper case S.

**Review by jamezp — CHANGES_REQUESTED** [review]
I'm not really sure I understand the use case for this change. Reading the JIRA and associated JIRA's I didn't get out of it that we should be logging a message like this at a INFO message.

**Comment by darranl** [comment]
@scottmarlow FYI this one is awaiting a response for the review from @jamezp

**Comment by bstansberry** [comment]
@scottmarlow If you'd like this in, please adjust the capitalization. Thanks.

**Comment by scottmarlow** [comment]
I somehow missed the feedback from last summer.  Correcting now.

File: jpa/hibernate5_3/src/main/java/org/jboss/as/jpa/hibernate5/JpaLogger.java (hunk 1 above)
Comment (scottmarlow) [reply jpa/hibernate5_3/src/main/java/org/jboss/as/jpa/hibernate5/JpaLogger.java:48]: Thanks James!

**Comment by scottmarlow** [comment]
> @scottmarlow If you'd like this in, please adjust the capitalization. Thanks.

@bstansberry just did that, thanks!

**Comment by scottmarlow** [comment]
> I'm not really sure I understand the use case for this change. Reading the JIRA and associated JIRA's I didn't get out of it that we should be logging a message like this at a INFO message.

@jamezp the idea is to make it more obvious when the second level cache is enabled, since the second level caching comes with a cost and applications may not know that it is enabled.

**Pull request merged**